phoenix-android/src/main/res/values-sk/strings.xml

```diff
@@ -0,0 +1,773 @@
+
+    <!-- mutual close -->
+
+    <string name="mutualclose_title">Vyprázdniť peňaženku</string>
+    <string name="mutualclose_loading">Načítavanie…</string>
+    <string name="mutualclose_checking_balance">Kontrolujem zostatok…</string>
+    <string name="mutualclose_balance">Zostatok: <b>%1$s</b> (≈ %2$s)</string>
+    <string name="mutualclose_no_channels">Peňaženka nemá žiadne kanály, ktoré by bolo možné zavrieť.</string>
+    <string name="mutualclose_button">Vyprázdniť peňaženku</string>
+    <string name="mutualclose_error_chain_mismatch">Táto adresa používa iný blockchain</string>
+    <string name="mutualclose_error_chain_reqparams">Táto adresa používa nepodporované funkcie</string>
+    <string name="mutualclose_error_chain_generic">Táto adresa nie je podporovaná</string>
+    <string name="mutualclose_done">Počet zatvorených kanálov: %1$d. Podrobnosti nájdete na domovskej obrazovke.</string>
+
+    <string name="forceclose_no_channels">Žiadne kanály na zatvorenie.</string>
+
+    <!-- channels view -->
+
+    <string name="channelsview_title">Platobné kanály</string>
+    <string name="channelsview_menu_import_channels">Importovať kanály</string>
+    <string name="channelsview_header">Prehľad</string>
+    <string name="channelsview_balance">Zostatok</string>
+    <string name="channelsview_balance_about"><b>Zostatok</b>: Toto je súhrnný zostatok vašich aktívnych kanálov. Toľko môžete poslať cez Lightning.</string>
+    <string name="channelsview_inbound">Prichádzajúca likvidita</string>
+    <string name="channelsview_inbound_about"><b>Prichádzajúca likvidita</b> je to, čo vaše kanály môžu prijať cez Lightning, bez toho, aby museli prejsť na on-chain a platiť poplatky.</string>
+    <string name="channelsview_loading_channels">Načítavanie údajov o kanáloch…</string>
+    <string name="channelsview_no_channels">Zatiaľ nemáte žiadne kanály.\n\nNový platobný kanál sa otvorí automaticky, keď to bude potrebné.</string>
+
+    <!-- channels details -->
+
+    <string name="channeldetails_title">Podrobnosti kanála</string>
+    <string name="channeldetails_not_found">Pre tento identifikátor neexistuje žiadny aktívny kanál.</string>
+    <string name="channeldetails_channel_id">ID kanála</string>
+    <string name="channeldetails_state">Stav</string>
+    <string name="channeldetails_spendable">Zostatok</string>
+    <string name="channeldetails_receivable">Likvidita na prichádzajúce platby</string>
+
+    <string name="channeldetails_commitments">Aktívne záväzky</string>
+    <string name="channeldetails_inactive_commitments">Neaktívne záväzky</string>
+    <string name="channeldetails_commitment_funding_tx_id">Financujúca tx:</string>
+    <string name="channeldetails_commitment_balance">Zostatok:</string>
+    <string name="channeldetails_commitment_funding_capacity">Kapacita:</string>
+    <string name="channeldetails_commitment_linked_payments">Vyvolané:</string>
+
+    <string name="channeldetails_json">Zobraziť holé dáta</string>
```

Comment:
Be consistent: Zobraziť holé dáta vs. Importovať surové údaje kanála

Reply:
Very cool. Make pull request with the changes if you can.

Reply:
Creating pull request with these changes, thank you!